ursula_cli/shell.py

```diff
@@ -527,6 +542,14 @@ def parse_args():
         '--heat-stack-name', default=None,
         help='Name of the heat stack when heat provisioner is used',
     )
+    parser.add_argument(
+        '--heat-stack-update', default=False, action='store_true',
+        help='Update the heat stack',
+    )
+    parser.add_argument(
+        '--heat-parameters', default=None,
+        help='Semicolon separated list of KEY=VALUE parameters for heat',
```

Comment:
the heatclient cli allows you to do this:

    heat -P thing=stuff -P something=else

or this:

  heat -P thing=stuff;something=else

so format_paramters uses a semicolon to split up the key/value pairs if there's only one -P supplied, which is the behaviour I've used here.